e2e/stories/select-record.e2e.ts

```diff
@@ -384,6 +388,112 @@ describe("filter records REST API", async () => {
     test(testData, successTests, failTests);
   });
 
+  describe("when filtering date related types", () => {
+    type MinMax = { min: number, max: number };
+    const fakeNumber = (options: MinMax) => faker.random.number(options);
+    const fakePastDatetime = (yearsAgo: MinMax) => faker.date.past(fakeNumber(yearsAgo)).toISOString();
+    const fakeFutureDatetime = (yearsFromNow: MinMax) => faker.date.future(fakeNumber(yearsFromNow)).toISOString();
+
+    function testDate({
+      fieldName,
+      fakePastDate,
+      fakeFutureDate
+    }) {
+      const testData = [
+        { [fieldName]: null },
+        { [fieldName]: fakePastDate({ min: 11, max: 30 }) },
+        { [fieldName]: fakePastDate({ min: 0, max: 10 }) },
+        { [fieldName]: fakeFutureDate({ min: 0, max: 10 }) },
+        { [fieldName]: fakeFutureDate({ min: 11, max: 30 }) },
+      ];
+
+      const [, firstDate, secondDate, thirdDate, lastDate] = testData.map((t) => t[fieldName]);
+
+      const successTests = [
+        {
+          title: "isEqualTo",
+          condition: field(fieldName).isEqualTo(secondDate),
+          expectedLength: 1,
```

Comment:
What is bothering me in these tests is that you are testing the length of results, but not that this result is expected, do you know what I mean? 
It can actually return the correct amount of records, but the results will be wrong and we do not get it.
What do you think?

Reply:
yeah, I agree. that was the way quickest way I found to create those e2e tests. I can improve these ones :+1: